armi/operators/tests/test_operators.py

```diff
@@ -156,6 +221,20 @@ def test_snapshotRequest(self, fakeDirList, fakeCopy):
             self.assertTrue(os.path.exists("snapShot0_2"))
 
 
+class TestCreateOperator(unittest.TestCase):
+    def test_createOperator(self):
+        """Test that an operator can be created from settings.
+
+        .. test:: Create an operator from settings.
+            :id: T_ARMI_OPERATOR_SETTINGS
+            :tests: R_ARMI_OPERATOR_SETTINGS
+        """
+        cs = settings.Settings()
+        o = Operator(cs)
+        self.assertTrue(isinstance(o, Operator))
+        self.assertTrue(isinstance(o.cs, settings.Settings))
```

Comment:
It might also be easy to test here that the operator _depends_ on the settings, which is called out in the acceptance criteria.

Reply:
Frankly, there isn't much to an operator, except a bunch of properties that directly call settings.

But I added some testing to this test to validate `burnSteps` and `powerFractions` values in the operator match what came from the setings.